### PR TITLE
[Blogspot] Add exclusion (fix #8113)

### LINKED ISSUE #8113: Blogspot.com blogs breaks video stream
http://streamfare.com/fox_news_2.html
http://streamfare.com/univisioni.html
http://streamfare.com/2abc.html
etc

`Blogspot.com blogs` breaks video.

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/Blogspot.com_blogs.xml [hunk 1]
```diff
@@ -150,6 +218,79 @@
...
+		<test url="http://sewkindofwonderful.blogspot.si/" />
+		<test url="http://sewkindofwonderful.blogspot.sk/" />
+		<test url="http://sewkindofwonderful.blogspot.sn/" />
+		<test url="http://sewkindofwonderful.blogspot.tw/" />
+		<test url="http://sewkindofwonderful.blogspot.ug/" />
+
+		<!-- Causes MCB on some pages such as http://streamfare.com/telemundo.html.
+			See https://github.com/EFForg/https-everywhere/issues/8113 -->
+		<exclusion pattern="^http://2livewatch\.blogspot\.com/" />
+			<test url="http://2livewatch.blogspot.com/2016/10/telemundo-222.html" />
```
Comment (J0WI) [inline src/chrome/content/rules/Blogspot.com_blogs.xml]: Why not limiting the exclusion to this specific post?

Comment (Bisaloo) [reply src/chrome/content/rules/Blogspot.com_blogs.xml]: Not sure what you mean. It probably causes MCB on other paths from the same website and possibly elsewhere as well.

Comment (J0WI) [reply src/chrome/content/rules/Blogspot.com_blogs.xml]: I didn't found any other posts.

File: src/chrome/content/rules/Blogspot.com_blogs.xml [hunk 2]
```diff
@@ -288,7 +288,7 @@
 
 		<!-- Causes MCB on some pages such as http://streamfare.com/telemundo.html.
 			See https://github.com/EFForg/https-everywhere/issues/8113 -->
-		<exclusion pattern="^http://2livewatch\.blogspot\.com/" />
+		<exclusion pattern="^http://2livewatch\.blogspot\.com/2016/10/telemundo-222.html" />
```
Comment (J0WI) [inline src/chrome/content/rules/Blogspot.com_blogs.xml]: Unescaped `.` in `.html`

**Pull request merged**